src/frontend/calendar/index.js

```diff
@@ -14,27 +14,47 @@ import { EventsProvider } from './event-context';
 
 function Calendar( { events } ) {
 	const today = new Date();
-	const [ { month, year }, setDate ] = useState( {
-		month: today.getMonth(),
-		year: today.getFullYear(),
-	} );
+	const currentMonth = today.getMonth();
+	const currentYear = today.getFullYear();
+	const currentMonthYear = {
+		month: currentMonth,
+		year: currentYear,
+	};
+	const [ { month, year }, setDate ] = useState( currentMonthYear );
 
 	return (
 		<EventsProvider value={ events }>
 			<div className="wporg-meeting-calendar__header">
-				<Button
-					onClick={ () => void setDate( { month: month - 1, year } ) }
-				>
-					{ __( 'Previous', 'wordcamporg' ) }
-				</Button>
-				<h2 aria-live="polite" aria-atomic>
-					{ date( 'F Y', new Date( year, month, 1 ) ) }
-				</h2>
-				<Button
-					onClick={ () => void setDate( { month: month + 1, year } ) }
-				>
-					{ __( 'Next', 'wordcamporg' ) }
-				</Button>
+				<div className="wporg-meeting-calendar__btn-group">
+					<Button onClick={ () => void setDate( currentMonthYear ) }>
+						{ __( 'Today', 'wporg' ) }
+					</Button>
+					<Button
+						onClick={ () =>
+							void setDate( { month: month - 1, year } )
+						}
+					>
+						{ __( 'Previous', 'wporg' ) }
+					</Button>
+					<Button
+						onClick={ () =>
+							void setDate( { month: month + 1, year } )
+						}
+					>
+						{ __( 'Next', 'wporg' ) }
+					</Button>
+				</div>
+				<div>
+					<h2 aria-live="polite" aria-atomic>
+						{ date( 'F Y', new Date( year, month, 1 ) ) }
+					</h2>
+				</div>
+				<div>
+					<ButtonGroup>
```

Comment:
`ButtonGroup` returns a div, so you don't need the extra div wrapper (a minor issue here, but excessive nesting of elements can be a performance hit that's noticeable on screen readers)

Reply:
_(Fixed this myself so I could merge the PR)_